### DISCUSSION TITLE
DBConvert Streams operating system/ architecture.

### DISCUSSION THREAD

**Discussion by slotix** [slotix/dbconvert-streams-public#33] [Polls]
We value your feedback on DBConvert Streams! Please take a moment to let us know about your experience by answering the following question.

Thank you for participating in our poll! Your feedback is invaluable to us.

**Comment by mstephano** [comment]
Mac OS X could also be split as AMD64 and ARM64. I am on ARM64 (Mac M1).

**Reply by slotix** [reply]
Updated Poll
